src/cata_tiles.h

```diff
@@ -716,6 +716,7 @@ class cata_tiles
 
         tripoint bul_pos;
         std::string bul_id;
+        int bul_rotation;
```

Comment:
C++ likes to leave basic fields as uninitialized mess, this prevents potential bugs
```suggestion
        int bul_rotation = 0;
```